src/app/project/beta/ontology-classes/ontology-class-item/ontology-class-item.component.ts

```diff
@@ -71,14 +72,9 @@ export class OntologyClassItemComponent implements OnInit {
         this._router.navigateByUrl(route);
     }
 
-    /**
-     * given a Html element, compare the scrollHeight and the clientHeight
-     *
-     * @param elem the element which has the line-clamp css
-     * @returns inverse of comparison between the scrollHeight and the clientHeight of elem
-     */
-    compareElementHeights(elem: HTMLElement): boolean {
-        return !(elem.scrollHeight > elem.clientHeight);
+
+    trimLabel(fullString: string) {
```

Comment:
Interesting! I suppose since the width is fixed, this isn't a problem with responsiveness.